<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NicknameDialog</class>
 <widget class="QDialog" name="NicknameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your nickname.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNickName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ratingSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelRange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter initial rating, range 1000-3000.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
